components/Form/index.jsx

```diff
@@ -1,36 +1,39 @@
-import React from "react";
-import { TextInput } from "../NameInput";
-import { GoogleExternalSignup, AppleExternalSignup } from "./ExternalSignup";
+import React from 'react'
+import { TextInput } from '../NameInput'
+import { GoogleExternalSignup, AppleExternalSignup } from './ExternalSignup'
+import { Button } from '../Button'
 
 export const Form = ({ login, signup }) => {
-	return (
-		<form className="bg-white rounded-lg p-5 mx-4 md:mx-auto flex flex-col justify-between gap-5 drop-shadow-xl">
-			<div className="signup-with flex justify-between">
-				<GoogleExternalSignup />
-				<AppleExternalSignup />
-			</div>
+  return (
+    <form className='bg-white rounded-lg p-5 mx-4 md:mx-auto flex flex-col justify-between gap-5 drop-shadow-xl'>
+      <div className='signup-with flex justify-between'>
+        <GoogleExternalSignup />
+        <AppleExternalSignup />
+      </div>
 
-			<div className="relative flex py-2 items-center">
-				<div className="flex-grow border-t border-gray-300"></div>
-				<span className="flex-shrink mx-4 text-gray-600">OR</span>
-				<div className="flex-grow border-t border-gray-300"></div>
-			</div>
+      <div className='relative flex py-2 items-center'>
```

Comment:
Is there a particular reason you prefer single code to double quote?
Or it's for consistency

Reply:
My prettier/es-lint settings auto change to single quote. I don't mind either way